org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -939,11 +977,38 @@ protected virtual void ModifyTargetPose(ref MixedRealityTransform targetPose, re
 
             bool applySmoothing = ShouldSmooth && smoothingLogic != null;
 
-            targetPose.Position = (applySmoothing && !useForces) ? smoothingLogic.SmoothPosition(HostTransform.position, targetPose.Position, moveLerpTime, Time.deltaTime) : targetPose.Position;
-            targetPose.Rotation = (applySmoothing && !useForces) ? smoothingLogic.SmoothRotation(HostTransform.rotation, targetPose.Rotation, rotateLerpTime, Time.deltaTime) : targetPose.Rotation;
+            targetPose.Position = (applySmoothing && !UseForces) ? smoothingLogic.SmoothPosition(HostTransform.position, targetPose.Position, moveLerpTime, Time.deltaTime) : targetPose.Position;
+            targetPose.Rotation = (applySmoothing && !UseForces) ? smoothingLogic.SmoothRotation(HostTransform.rotation, targetPose.Rotation, rotateLerpTime, Time.deltaTime) : targetPose.Rotation;
             targetPose.Scale = applySmoothing ? smoothingLogic.SmoothScale(HostTransform.localScale, targetPose.Scale, scaleLerpTime, Time.deltaTime) : targetPose.Scale;
         }
 
+        private void UpdateRigidbodyMovementType()
+        {
+            // Find the most recent interactor that has a movement type override, if any, using reverse order of selecting interactors.
```

Comment:
Just a question.... does the reverse order mean that the override is controlled by the last selecting interactor?

Reply:
Yes.  And that matches XRI's XRGrabInteractable's selection of an override.